### PR TITLE
[15.0][MIG] sale force invoiced

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration
@ForgeFlow

### CODE REVIEW

**Review by ChrisOForgeFlow — APPROVED** [review]
+1

**Review by HaraldPanten — CHANGES_REQUESTED** [review]
**Functional review**

Steps to reproduce:

After activating Locking options for Sale orders:

1- Create a quotation and confirm it.
2- You have a locked sale order.
3- Unlock the SO.
4- Edit and mark the checkbox for forcing the invoice status to fully invoiced.

Shouldn't the checkbox be available to be edited only when SO is locked?

**Comment by LoisRForgeFlow** [comment]
> **Functional review**
> 
> Steps to reproduce:
> 
> After activating Locking options for Sale orders:
> 
> 1- Create a quotation and confirm it. 2- You have a locked sale order. 3- Unlock the SO. 4- Edit and mark the checkbox for forcing the invoice status to fully invoiced.
> 
> Shouldn't the checkbox be available to be edited only when SO is locked?

Hi @HaraldPanten Thas was changed here https://github.com/OCA/sale-workflow/pull/1823/commits/4e799dc60bf6f66407d89325aee39201b18c49aa. Since locking a SO is a manual action, not all companies lock them and just let them in sales status, thefore instead of forcing them to lock SOs, we preferred to add more flexibility to the functionality.

Does this makes sense to you?

File: sale_force_invoiced/README.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+   :local:
+
+Usage
+=====
+
+#. Create a sales order and confirm it.
+#. Deliver the products/services.
+#. Create an invoice and reduce the invoiced quantity. The sales order
+   invoicing status is 'To Invoice'.
+#. Lock the Sale Order, to change the status of it to 'Done'.
```
Comment (LoisRForgeFlow) [inline sale_force_invoiced/README.rst]: @HaraldPanten maybe what is missing is to update the usage?

**Comment by HaraldPanten** [comment]
Hi @LoisRForgeFlow Definitely, I didn't keep in mind that commit... you're right.

@JasminSForgeFlow just update the Readme as Lois is suggesting and I'll approve it.

THX.

**Comment by JasminSForgeFlow** [comment]
Hi @HaraldPanten ,

Usage has been update.

Thanks

**Review by HaraldPanten — APPROVED** [review]
Functional review, LGTM

**Comment by LoisRForgeFlow** [comment]
@HaraldPanten you can merge at your convenience!

**Pull request merged**